runner/src/config.rs

```diff
@@ -25,6 +25,7 @@ pub struct Config {
     pub vcpu: VCpu,
     #[serde(default)]
     pub platform: Platform,
+    pub benchmark: Option<bool>,
```

Comment:
I am thinking that we might need more options to configure the benchmarks later on, so we probably want to create a new table rather than just a single field (for instance to choose what to benchmark).

To that that you would need to create a `Benchmark` struct, add `#[derive(Deserialize, Debug, Default)]`
and `#[serde(deny_unknown_fields)]` and then add one field (for instance `enable` to toggle benchmarks). You can take inspiration from the other tables such as `Log` or `Debug`.

Then in the `config.toml` it will look like that:

```toml
[benchmark]
enable = true
```

And that gives us more room to expend with other fields later on :)